src/util/base58.rs

```diff
@@ -45,9 +48,10 @@ impl fmt::Display for Error {
             Error::BadByte(b) => write!(f, "invalid base58 character 0x{:x}", b),
             Error::BadChecksum(exp, actual) => write!(f, "base58ck checksum 0x{:x} does not match expected 0x{:x}", actual, exp),
             Error::InvalidLength(ell) => write!(f, "length {} invalid for this base58 type", ell),
-            Error::InvalidVersion(ref v) => write!(f, "version {:?} invalid for this base58 type", v),
+            Error::InvalidAddressVersion(ref v) => write!(f, "address version {:?} invalid for this base58 type", v),
```

Comment:
```suggestion
            Error::InvalidAddressVersion(ref v) => write!(f, "address version {} is invalid for this base58 type", v),
```

Here we can normally print version number

Reply:
done in 04a14b91e4f8fd81a8d218c27e04789d00a86e6b